### PR TITLE
chore: add a contribution note to the readme

### PR DESCRIPTION
**Pull request by Foxandxss**
I want to add this note because in the state we are with the project right now, we cannot allow people to waste their time making PRs.

We make lot of decisions everyday, we change lots of code too, and I feel bad if I have to close someones PR because it is not the way we want to implement it and we cannot afford to walk that person PR in the way we want it.

As soon as this volatile state disappear, well, I am more than happy to get all PRs possible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,4 +44,7 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
 
 ## Contributing to the project
 
+**NOTE: We appreciate all the interest and enthusiasm, but this project is just in such a volatile state right now that we are not going to accept pull requests that adds new component or features.
+We are in constant discussion about how can we implement certain features that we don't want yo waste your time on PR that are not going to be merged.**
```
Comment (icfantv) [inline README.md]: change "yo" to "to"

Comment (icfantv) [inline README.md]: change PR to PRs

Comment (icfantv) [inline README.md]: "adds" to "add"

File: README.md [hunk 2]
```diff
@@ -44,4 +44,7 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
 
 ## Contributing to the project
 
+**NOTE: We appreciate all the interest and enthusiasm, but this project is just in such a volatile state right now that we are not going to accept pull requests that adds new component or features.
```
Comment (icfantv) [inline README.md]: "...not going to accept pull requests that add new components...".  sorry, i accidentally committed the previous comment before the grammar registered.

File: README.md [hunk 3]
```diff
@@ -44,4 +44,7 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
 
 ## Contributing to the project
 
+**NOTE: We appreciate all the interest and enthusiasm, but this project is just in such a volatile state right now that we are not going to accept pull requests that adds new components or features.
```
Comment (icfantv) [inline README.md]: change 'adds' to 'add'

File: README.md [hunk 4]
```diff
@@ -44,4 +44,7 @@ See [this](https://github.com/angular/angular/blob/master/README.md) for up-to-d
...
+We are in constant discussion about how can we implement certain features that we don't want to waste your time on PRs that are not going to be merged.**
```
Comment (icfantv) [inline README.md]: "...certain features and we don't want you to waste your time on PRs that..."

**Comment by pkozlowski-opensource** [comment]
@Foxandxss I don't think we should be refusing contributions like this. We can add a very explicit note that people interested in helping out should discuss things first.

Personally I was many times surprised by people who were coming "out of nowhere", figured everything out by themselves and did "right thing". Most of those people then became core contributors. We need to seek to invite / encourage those people not to discourage them up-front.

Sure, it comes at the cost of potentially dealing with "bad" PRs but IMO this is price worth paying. Sure, we need to discuss many things but we might have interesting insights from anyone.

**Comment by Foxandxss** [comment]
No, it is not about discouraging people up-front, it is about not wasting people's time.

Say someone tries to create a datepicker now and creates a PR with a complex datepicker, lots of tests... But... it is not the way we wanted to create it (with a engine that could be exchanged for another one).

In that case, we can say: "Thank you, won't merge", or we can merge it and fix it later (bad) or we can walk the guy in how we want to do it (bad too). We can get the PR and fix it locally before merge (bad too).

In other cases, it won't really matter, but everything is so volatile right now, we are giving it too much thought in how to do something, that I think that preventing people right now to write new components and features is the right thing to do. It is not discouraging again, it is not waste their time with a PR that won't be merged nor waste our time walking that person in... no, please change this to that and rewrite this again... uh, you have to change it yet again.

**Comment by pkozlowski-opensource** [comment]
> it is about not wasting people's time.

Then I think we should say just this: "Hey, it is moving target right now, so we can't guarantee that your that your PR will get merged". 

People are grown up, they should know that there is no guarantees in life (err, open source). We don't need to protect people from themselves. 

Personally I would prefer "wasting" time of 10 people if it brings 1 core contributor.

We might have a warning in the readme but the current phrasing sounds too discouraging to me (once again, this might be just my personal perception). On top of this I'm not sure people who can't figure out things on their own are not going to read readme anyway :-)

**Comment by Foxandxss** [comment]
> Then I think we should say just this: "Hey, it is moving target right now, so we can't guarantee that your that your PR will get merged".

I can do some rephrasing yes.

> Personally I would prefer "wasting" time of 10 people if it brings 1 core contributor.

Yeah, but don't think people want that :P

>  On top of this I'm not sure people who can't figure out things on their own are not going to read readme anyway :-)

That is not my problem heh.

**Comment by Foxandxss** [comment]
Reworded.

**Comment by pkozlowski-opensource** [comment]
LGTM

**Pull request closed without merging**